projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-flag/components/update-flag/update-flag.component.spec.ts

```diff
@@ -434,4 +435,30 @@ describe('UpdateFlagComponent', () => {
     errorMessageElement = fixture.debugElement.nativeElement.querySelector('.govuk-error-message');
     expect(errorMessageElement).toBeNull();
   });
+
+  it('should display correct title based on the display mode', () => {
+    component.displayContextParameter = '#ARGUMENT(UPDATE)';
+    component.setUpdateCaseFlagTitle(activeFlag);
+    console.log('UPDATE FLAG TITLE', component.updateFlagTitle);
```

Comment:
@johnbenjamin-hmcts - this `console.log` statement can be removed

Reply:
@Daniel-Lam - thank you, I have removed this console.log statement